Bottom: 8c12e00e16e7ee8ecff7bb4fba7758d4112ddf25
Top:    5dc57669cfe5e21c557997e73028089331e468dc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:46:05 +0000

Refresh of use-calling-wrappers-rather

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 663bdf3..5847567 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -497,7 +497,7 @@ common_fill_pcb_pad (hidGC gc, PadType *pad, bool clear, bool mask)
         {
           Coord x[4], y[4];
           common_get_pad_polygon (x, y, pad, w);
-          hid_fill_polygon (gc, 4, x, y);
+          hid_draw_fill_polygon (gc, 4, x, y);
         }
       else
         hid_draw_line (gc, pad->Point1.X, pad->Point1.Y,
